packages/contentful--create-contentful-app/src/index.ts

```diff
@@ -126,7 +127,15 @@ async function initProject(appName: string, options: CLIOptions) {
     await cloneTemplateIn(fullAppFolder, templateSource);
 
     if (!isInteractive && isContentfulTemplate(templateSource) && normalizedOptions.action) {
-      cloneAppAction(fullAppFolder, !!normalizedOptions.typescript);
+      await cloneAppAction(fullAppFolder, !!normalizedOptions.typescript);
+    }
+
+    if (
+      !isInteractive &&
+      isContentfulTemplate(templateSource) &&
+      normalizedOptions.deliveryFunction
+    ) {
+      await cloneDeliveryFunction(fullAppFolder, !!normalizedOptions.typescript);
```

Comment:
parallelize here too, with the action script above

Reply:
Personally I feel that in this instance the gained performance does not weigh up against the increased code complexity of parallelising these two tasks. As the consumer of the library, since I'm only calling this function once in the entire lifecycle of my application, I'd rather wait a few more seconds and have the logs read nicely step by step.